source/codegen/metadata/nidaqmx/functions.py

```diff
@@ -7283,14 +7282,133 @@
             },
             {
                 'callback_params': [
+                    {
```

Comment:
`RegisterDoneEvent` has two interesting callback params:

`TaskHandle`: typically not part of a response message. We're tracking context through the output stream, so there's no reason to include it in the response. So: `include_in_proto: False`

'status': We auto-add `status` to responses and assume that it is the output of the method. That's true for `Register` itself (which can fail, in which case we early return). But when you're called as a callback, you don't automatically have a `status` output to use. `RegisterDoneEvent` *does* have a status, which we'll use. The other ones *don't* but still have a `default_status_param`.